### DISCUSSION TITLE
Signia still in active development?

### DISCUSSION THREAD

**Discussion by justinfernald** [tldraw/signia#85] [General]
Just checking to see if Signia is still in active development. I see that it was not too long since the last bit of activity, but I just want to make sure this library is still being developed on so I can use in a project.

**Comment by navaru** [comment]
[This PR](https://github.com/tldraw/tldraw/pull/1620) incorporated Signia into tldraw. Active development happens here [tldraw/state/lib/core](https://github.com/tldraw/tldraw/tree/main/packages/state/src/lib/core).
